src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+  const declarations = sourceString.split(';');
+
+  declarations.forEach((declaration) => {
+    if (declaration.trim() !== '') {
+      const [key, value] = declaration.split(':');
+
+      styleObject[key.trim()] = value.trim();
+    }
+  });
```

Comment:
[CODE PRACTICE] - Instead of using a `forEach` loop to populate `styleObject`, consider using `reduce` to transform the array of declarations directly into an object. This approach aligns with the guideline of avoiding loops and utilizing iteration methods effectively.